core/src/components/header/header.tsx

```diff
@@ -46,6 +46,12 @@ export class Header implements ComponentInterface {
    */
   @Prop() collapse?: 'condense' | 'fade';
 
+  /**
+   * If `true`, the header will have a line at the bottom.
+   * TODO(ROU-10855): add support for this prop on ios/md themes
+   */
```

Comment:
Since you are reflecting a class based on the prop and using that as the CSS selector, the attribute does not need to be reflected.